src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,121 @@
+package duke.ui;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import duke.task.Task;
+
+/**
+ * Prints messages with desired formatting.
+ *
+ * @author Lian Kok Hai
+ */
+public class Ui {
+    private int maxCharPerLine = 100;
+    private String horizontal;
```

Comment:
Maybe you could consider naming this variable differently to make it more clear?